KakapoExample/KakapoTests/KakapoDBTests.swift

```diff
@@ -361,8 +343,8 @@ class KakapoDBTests: QuickSpec {
                 let users = sut.create(UserFactory.self, number: 100)
                 
                 dispatch_apply(100, dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0)) { i in
-                    sut.update(users[i])
-                    sut.delete(users[i])
+                    let _ = try? sut.update(users[i])
```

Comment:
For the optional binding you need to get the possible error it throws, otherwise we get compiler warning. But probably better to force unwrap when we are sure, yes